pkg/helper/request.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Please move this file to test/e2e because it only be used by E2E testing.

Reply:
OK, I have moved it to `test/e2e/helper`.